static-build/cloud-hypervisor/docker-build/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM ubuntu:18.04
+
+RUN apt-get update
+RUN apt-get install -yq build-essential mtools libssl-dev pkg-config curl git
+RUN nohup curl https://sh.rustup.rs -sSf | sh -s -- -y
```

Comment:
I'm just going to trust this is 'standard practice', but, curling source into a shell is generally a bit of a security 'no no'...

Reply:
Yeah that's not ideal, but that's the way rust is usually installed. BTW, just checked the official Rust webpage and they updated the instructions on how to install:
```
curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
```